### ISSUE TITLE
Statsd incorrect unit

### ISSUE THREAD

**Issue by NatMarchand** [containous/traefik#5944]
### What did you do?

When I configure statsd provider for tracing and I trace an HTTP request which is known to last approximately 1 second.

### What did you expect to see?
On my statsd receiver:
```
traefik.service.request.duration:1055.487|ms
traefik.entrypoint.request.duration:1058.461|ms
```
See statsd spec https://github.com/b/statsd_spec#timers explains that the timers are milliseconds.

### What did you see instead?
```
traefik.service.request.duration:1.055487|ms
traefik.entrypoint.request.duration:1.058461|ms
```
### Output of `traefik version`: (_What version of Traefik are you using?_)
```
Version:      2.0.1
Codename:     montdor
Go version:   go1.13.1
Built:        2019-09-26T16:20:36Z
OS/Arch:      windows/amd64
```

### Additional infos
It seems that there was issues with go-kit/kit which is used to handle metrics, see https://github.com/go-kit/kit/issues/608 and https://github.com/go-kit/kit/pull/610

**Comment by pascalandy** [comment]
Can you please share your statsd provider's configuration?
Thank you

**Comment by NatMarchand** [comment]
Sure! Nothing fancy

```yaml
metrics:
  statsD:
    address: 127.0.0.1:8125
    pushInterval: 5s
```

**Comment by traefiker** [comment]
Closed by #6054.
